core-chart/templates/argocd/setup.yaml

```diff
@@ -156,9 +185,10 @@ spec:
         - name: script
           mountPath: /mnt/script
           readOnly: true
+      
       volumes:
       - name: script
         configMap:
           name: "{{ .Release.Name }}-argocd-kubestellar-setup"
           defaultMode: 0744
-{{- end }}
+{{- end }}
```

Comment:
Please do not remove the newline at the end of the file.
<img width="248" height="116" alt="image" src="https://github.com/user-attachments/assets/bcccc2de-f771-4cb4-8ef4-ebee47b7c6d0" />